src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml

```diff
@@ -19,7 +19,7 @@
         }
 
         //Build Editor for Content Item
-        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, false, "", Model.PrefixValue);
+        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, true, "", Model.PrefixValue);
```

Comment:
This is for existing content and isnew should be false

Reply:
Could you look into what the correct fix is Niraj, if this is not it

Reply:
```suggestion

        //AJAX request is new request and will not have CollectionShape.
        var isNew = Model.CollectionShape == null ? true : false ;
        dynamic contentItemEditor = await ContentItemDisplayManager.BuildEditorAsync(contentItem, updater, isNew, "", Model.PrefixValue);
```

Reply:
All new widgets are added using ajax call. In ajax call it doesn't have CollectionShape